docs/deploy.md

```diff
@@ -146,6 +184,17 @@ For the official homeworld servers:
  * Log into the server directly with your SSH keys
    - Verify the host keys based on the text printed before the login console
 
+For example, ``ssh root@egg-sandwich.mit.edu``. You might need to remove previous SSH hostkeys from known_hosts if you've set up the cluster before.
```

Comment:
Maybe this should be more clearly associated with the instruction that it's an example for?